### PR TITLE
reqWrapper as Readable stream

### PR DESCRIPTION
**Pull request by schamberg97**
This is a breaking change that reimplements the `reqWrapper` as a Readable stream. The wrapper creates empty `req.body` object, BUT no longer deals with populating it. Native HTTP module also does not populate `req.body` by default.

### CODE REVIEW

**Comment by schamberg97** [comment]
Additionally, I have removed a check that request body is not processed on `GET` methods. While this is contrary to the standards, sometimes request bodies are erroneously sent with `GET` requests. I do not have a strong opinion on this matter, so feel free to revert that change

File: tests/0http-integration.spec.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 const expect = require('chai').expect
 const request = require('supertest')
 const { createReadStream } = require('fs')
+var jsonParser = require('body-parser').json
```
Comment (jkyberneees) [inline tests/0http-integration.spec.js:5]: const?

File: tests/0http-integration.spec.js [hunk 2]
```diff
@@ -29,8 +30,16 @@ describe('0http Web Framework - Low HTTP Server Integration', () => {
...
+    router.post('/echo2', (req, res) => {
+      let stringified = JSON.stringify(req.body)
+      res.end(stringified)
+    })
+
+    router.use(jsonParser())
+
     router.post('/echo', (req, res) => {
-      res.end(req.body)
+      let stringified = JSON.stringify(req.body)
```
Comment (jkyberneees) [inline tests/0http-integration.spec.js:41]: const?

**Comment by jkyberneees** [comment]
LGTM !!!

**Pull request merged**